app/src/androidTest/kotlin/cz/covid19cz/erouska/tests/ActivationTest.kt

```diff
@@ -0,0 +1,77 @@
+package cz.covid19cz.erouska.tests
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.rule.ActivityTestRule
+import cz.covid19cz.erouska.R
+import cz.covid19cz.erouska.screenObject.*
+import cz.covid19cz.erouska.testRules.DisableAnimationsRule
+import cz.covid19cz.erouska.ui.main.MainActivity
+import org.awaitility.Awaitility
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.TimeUnit
+
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ActivationTest {
+    private val welcomeScreen = WelcomeScreen
+    private val bluetoothPermissionScreen = BluetoothPermissionScreen
+    private val phoneNumberScreen = PhoneNumberScreen
+    private val smsScreen = SMSScreen
+    private val finishActivation = FinishActivationScreen
+    private val homeScreen = HomeScreen
```

Comment:
Není potřeba si to ukládat do proměnné - když máš object, tak ho můžeš rovnou používat, např. `WelcomeScreen.howToWorkScreen()`